src/History/Storage/ORMStorage.php

```diff
@@ -50,7 +50,9 @@ public function find(mixed $id): ?ProcessedMessage
 
     public function filter(Specification $specification): Collection
     {
-        return new EntityResult($this->queryBuilderFor($specification));
+        return (new EntityResult($this->queryBuilderFor($specification)))
```

Comment:
@bendavies Do I need to disable output walkers in the other methods?

Reply:
as noted here:
https://github.com/zenstruck/messenger-monitor-bundle/issues/137#issuecomment-2737301191 this is also a problem:

https://github.com/zenstruck/messenger-monitor-bundle/blob/1.x/templates/history.html.twig#L134